src/components/FormSubmit/index.js

```diff
@@ -5,20 +5,12 @@ import * as formSubmitPropTypes from './formSubmitPropTypes';
 import CONST from '../../CONST';
 import isEnterWhileComposition from '../../libs/KeyboardShortcut/isEnterWhileComposition';
 
-// This is a wrapper component to handle the ENTER key press, and submit the form.
-class FormSubmit extends React.Component {
-    constructor(props) {
-        super(props);
-
-        this.submitForm = this.submitForm.bind(this);
-    }
-
+function FormSubmit(props) {
```

Comment:
let's use prop destructuring to follow guidelines: https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#destructuring